docs/using-gitbase/functions.md

```diff
@@ -17,6 +17,7 @@ To make some common tasks easier for the user, there are some functions to inter
 |`uast(blob, [lang, [xpath]]) blob`| returns a node array of UAST nodes in semantic mode.                                                          |
 |`uast_children(blob) blob`| returns a flattened array of the children UAST nodes from each one of the UAST nodes in the given array.              |
 |`uast_extract(blob, key) text array`| extracts information identified by the given key from the uast nodes.                                       |
+|`uast_imports(blob) text array`| get all imports given the specified UAST blob. |
```

Comment:
For consistency with the rest of the table, I would change `get` by `gets`, or even by `returns`.